src/TypeScript/Analyzer/JavaScriptAnalyzer.cs

```diff
@@ -133,17 +136,31 @@ internal async Task ExecuteAnalysis(string filePath, IIssueConsumer consumer, Ca
             }
         }
 
-        private async Task<IEslintBridgeClient> GetJavaScriptClient(CancellationToken token)
+        private async Task<IEslintBridgeClient> GetJavaScriptClient(CancellationToken cancellationToken)
         {
             var port = await eslintBridgeProcess.Start();
```

Comment:
Should the call to `Start()` be inside the lock too?

Scenario:
Initial analysis request has completed successfully, process on port 888, init-linter called successfully. Then:
1. Thread A: first analysis request: Start() returns port 888
2. process crashes
3. Thread B: second analysis request: Start() returns port 999
4. Thread A resumes - compares port result, decides (incorrectly) that init-linter call is not required.

Reply:
The call to init-linter has to be made on the port that the Start() returned, so in this use case, if thread A got 888 and that process has died, any call on thread A will fail -- either the init-linter or the analyze-js, since both will call 888.
So if the process died at any point after a thread already got hold of the port number, I think that thread is gonna crash either way and the user would have to reanalyze. WDYT?